src/components/model/ModelStatusRollup.svelte

```diff
@@ -0,0 +1,254 @@
+  let resourceLogStatus: ModelStatus = 'none';
+  let status: ModelStatus = 'none';
+
+  $: {
+    if (model) {
+      const {
+        refresh_activity_type_logs: activityLogs,
+        refresh_model_parameter_logs: parameterLogs,
+        refresh_resource_type_logs: resourceLogs,
+      } = model;
+
+      activityLog = activityLogs[0] ?? null;
+      parameterLog = parameterLogs[0] ?? null;
+      resourceLog = resourceLogs[0] ?? null;
+
+      if (activityLog) {
+        if (activityLog.success) {
+          activityLogStatus = 'complete';
+        } else {
+          activityLogStatus = 'error';
+        }
+      } else {
+        activityLogStatus = 'extracting';
+      }
+
+      if (parameterLog) {
+        if (parameterLog.success) {
+          parameterLogStatus = 'complete';
+        } else {
+          parameterLogStatus = 'error';
+        }
+      } else {
+        parameterLogStatus = 'extracting';
+      }
+
+      if (resourceLog) {
+        if (resourceLog.success) {
+          resourceLogStatus = 'complete';
+        } else {
+          resourceLogStatus = 'error';
+        }
+      } else {
+        resourceLogStatus = 'extracting';
+      }
+
+      if (activityLogStatus === 'error' || parameterLogStatus === 'error' || resourceLogStatus === 'error') {
+        status = 'error';
+      } else if (
+        activityLogStatus === 'complete' &&
+        parameterLogStatus === 'complete' &&
+        resourceLogStatus === 'complete'
+      ) {
+        status = 'complete';
+      } else if (
+        activityLogStatus === 'extracting' &&
+        parameterLogStatus === 'extracting' &&
+        resourceLogStatus === 'extracting'
+      ) {
+        status = 'extracting';
+      }
+    } else {
+      status = 'none';
+    }
+  }
+
+  function selectActivityLog() {
+    selectedLog = 'activity';
+    dispatch('select', activityLog);
+  }
+
+  function selectParameterLog() {
+    selectedLog = 'parameter';
+    dispatch('select', parameterLog);
+  }
+
+  function selectResourceLog() {
+    selectedLog = 'resource';
+    dispatch('select', resourceLog);
+  }
+</script>
+
+<div class="model-status-container">
+  {#if mode === 'rollup' || mode === 'iconOnly'}
+    <div class="model-status-rollup" class:icon-only={mode === 'iconOnly'}>
+      <ModelStatusIcon {showCompleteStatus} {status} tooltipContent={status && rollupTooltipMessages[status]} />
+      {#if mode === 'rollup'}
+        {#if status === 'extracting'}
+          Extracting
+        {:else if status === 'complete' && showCompleteStatus}
+          Extracted
+        {:else if status === 'error'}
+          Errors extracting
+        {/if}
+      {/if}
+    </div>
+  {:else}
+    <div class="model-status-logs-container" class:horizontal={flow === 'horizontal'}>
+      <button
+        disabled={!selectable}
+        class="model-status-button"
+        class:selected={selectedLog === 'activity'}
+        on:click={selectActivityLog}
+      >
+        <ModelStatusIcon
```

Comment:
Could we do an 8px gap here?
<img width="69" alt="image" src="https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/e0459400-ce0b-45ee-b7c7-b786aa949b0c">
so that it matches mock
![image](https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/c6197fac-d7c6-48ee-b419-e66b66b649f6)